docker/checkout.sh

```diff
@@ -24,5 +24,5 @@ if [[ -n ${1} ]]; then
       dir=${1}
 fi
 
-git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ${dir}/src/CARMAMsgs --branch CARMASystem_3.1.0 --depth 1
-git clone https://github.com/usdot-fhwa-stol/carma-utils.git ${dir}/src/CARMAUtils --branch CARMASystem_3.1.0 --depth 1
+git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ${dir}/src/CARMAMsgs --branch carma-system-3.6.0 --depth 1
+git clone https://github.com/usdot-fhwa-stol/carma-utils.git ${dir}/src/CARMAUtils --branch carma-system-3.6.0 --depth 1
```

Comment:
On this particular repo, I think you actually want to leave these unchanged. The two delphi repos have not been maintained and so rely on older message sets.